src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -48,18 +48,14 @@ public function isProxyCandidate(Definition $definition): bool
     /**
      * {@inheritdoc}
      */
-    public function getProxyFactoryCode(Definition $definition, $id, $factoryCode = null): string
+    public function getProxyFactoryCode(Definition $definition, string $id, string $factoryCode): string
```

Comment:
making the argument mandatory is a BC break we should prevent, let's make it `string $factoryCode = null`
would that be enough to revert the composer.json change?

Reply:
hum, nvm
but I'd still prefer not hard-breaking BC, how would that look like?

Reply:
Well, the parameter is mandatory in the interface and not providing it will result in an exception anyway, so I thought we might as well enforce the interface here.

Reply:
> but I'd still prefer not hard-breaking BC, how would that look like?

What do you mean by not breaking BC? Keep the ProxyManager bridge compatible with DI 4.4? In that case, I'd need revert all changes made to the bridge. But since bridge is pretty much just glue between DI and ProxyManager, I thought it might be okay if bridge and DI are kept at the same version.